dev/conductor/core/lib/src/state.dart

```diff
@@ -77,7 +66,7 @@ String presentState(pb.ConductorState state) {
   buffer.writeln('\tCurrent git HEAD: ${state.engine.currentGitHead}');
   buffer.writeln('\tPath to checkout: ${state.engine.checkoutPath}');
   buffer.writeln(
-      '\tPost-submit LUCI dashboard: ${luciConsoleLink(state.releaseChannel, 'engine')}');
+      '\tPost-submit LUCI dashboard: $luciConsoleLink');
```

Comment:
This one and the flutter one may need to stay the same for now. Dart-internal will have the engine dashboards only after the migration to recipes v2.

Reply:
We monitor engine tests through cocoon, do we need to be monitoring at the luci level also?